ddtrace/internal/telemetry/writer.py

```diff
@@ -222,6 +227,8 @@ def __init__(self, is_periodic=True, agentless=None):
             # Force app started for unit tests
             if _TelemetryConfig.FORCE_START:
                 self._app_started()
+            if _TelemetryConfig.LOG_COLLECTION_ENABLED:
+                getLogger("ddtrace").addHandler(DDTelemetryLogHandler(self))
```

Comment:
I never noticed before, but this + the `log = getLogger(__name__)` at the top should be `from ddtrace.internal.logger import get_logger` instead.

I bet it is using `getLogger` because of the circular import, but now that should be resolved with this change too?

Reply:
Would we ever re-initialize the `TelemetryWriter`?

Do we need to check if the handler already exists on the logger?

This is going to maintain a reference to `self` for the lifetime of the application, unless we remove the handler later on destruction of `self`.


The scenario I am thinking of, if there is ever `writer_1 = TelemetryWriter(); writer_2 = TelemetryWriter()` we are going to have two handlers added to the root `ddtrace` logger, and even if `writer_2` is meant to replace `writer_1` as being the only `TelemetryWriter`, because of the reference held by the handler and then the logger, we will have 2 TelemetryWriters instead.

Reply:
:+1: as long as the concern is considered, then all good from me.